lib/ansible/modules/database/postgresql/postgresql_privs.py

```diff
@@ -541,7 +557,9 @@ def main():
             port=dict(type='int', default=5432),
             unix_socket=dict(default='', aliases=['login_unix_socket']),
             login=dict(default='postgres', aliases=['login_user']),
-            password=dict(default='', aliases=['login_password'], no_log=True)
+            password=dict(default='', aliases=['login_password'], no_log=True),
+            ssl_mode=dict(default='disable'),
+            ssl_rootcert=dict(default='')
```

Comment:
Not sure I understand what you're saying?  This PR adds ssl_rootcert to both modules.  Seems like they should both default to either empty string or both default to None?